src/Components/Navbar.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
-
+import { Link } from 'react-router-dom';
 export default function Navbar() {
   return (
-    <nav className="navbar navbar-expand-lg navbar-light fixed-top shadow-sm" id="mainNav">
+    <nav className="navbar navbar-expand-lg navbar-light shadow-sm" id="mainNav">
```

Comment:
use map for navbar items, something as follows:

```
const navbarItems = ['About', 'Contact']

return (
       ....
       <div className="collapse navbar-collapse" id="navbarResponsive">
           <ul className="navbar-nav ms-auto me-4 my-3 my-lg-0">
               {navbarItems.map(x =>  <li className="nav-item"> <a className="nav-link me-lg-3" href="#features"> {x} </a> </li>)}
           </ul>
          ....
       </div>
    ...
);
```

Reply:
This will ensure that you don't have to duplicate the code again in-case you need to add one more navbar item. 

Challenge for you: Try to come-up with a way to accommodate the 'Sign In' option as well in this map with its current tag & styling (Hint: see what other ways can this map function be used)